runtime/drivers/github/repo.go

```diff
@@ -28,7 +28,7 @@ func (c *connection) Root() string {
 
 // ListRecursive implements drivers.RepoStore.
 func (c *connection) ListRecursive(ctx context.Context, instID, glob string) ([]string, error) {
-	fsRoot := os.DirFS(c.tempdir)
+	fsRoot := os.DirFS(c.projectdir)
```

Comment:
In addition to these changes `Root()` should also be `c.tempdir`

Reply:
updated as `c.projectdir` in `Root()`